x-pack/plugins/security/public/views/management/roles_grid/components/permission_denied/permission_denied.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiEmptyPrompt, EuiFlexGroup, EuiPageContent } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+
+export const PermissionDenied = () => (
```

Comment:
This permission denied screen looks a bit different than the one that we use for the "users list", is this version what we're supposed to be using now?

Reply:
@cchaos which version of this screen is more correct?

## Option 1
![image](https://user-images.githubusercontent.com/3493255/56285438-3a18ec80-60e5-11e9-8c11-23a975d6c654.png)

## Option 2
![image](https://user-images.githubusercontent.com/3493255/56285474-4ef58000-60e5-11e9-95ca-e6c1c41c9135.png)

Reply:
Option 1, if you can.

Reply:
Great, thanks!

Reply:
Is it possible to use a little more positive message:

You need permission to manage users

Contact your system administrator.

Reply:
I like it! I'll make that change

Reply:
![image](https://user-images.githubusercontent.com/3493255/56302085-0ea6f980-6107-11e9-9e8d-ab3a34040935.png)